src/components/DisplayList.tsx

```diff
@@ -1,10 +1,33 @@
-import React from "react";
+import React, { useState } from "react";
 
-export default function DisplayList(props: { list: Array<string> }) {
+interface IDisplayListRequiredProps {
+  list: Array<any>;
+}
+
+interface IDisplayListOptionalProps {
+  onClick: any;
+}
+
+interface IDisplayListProps
+  extends IDisplayListRequiredProps,
+    IDisplayListOptionalProps {}
+
+const defaultProps: IDisplayListOptionalProps = {
```

Comment:
Why is the onClick optional?

Reply:
It's not really. I was testing the `DisplayList` component and the `onClick` wasn't necessary so I added a default because I was lazy. I can update for that tho.